system/Cache/Handlers/RedisHandler.php

```diff
@@ -70,13 +70,13 @@ class RedisHandler implements CacheInterface
 
 	//--------------------------------------------------------------------
 
-	public function __construct($config)
+	public function __construct(array $config)
 	{
-		$this->prefix = $config->prefix ?: '';
+		$this->prefix = $config['prefix'] ?? '';
 
-		if (isset($config->redis))
+		if (!empty($config))
```

Comment:
I know it's nit-picky, but according to our style guide, there should be a space after the exclamation.